src/components/menu/menu.scss

```diff
@@ -12,7 +12,7 @@ $menu-item-background-color: #ebebeb;
 
 .menu__trigger {
     border-color: transparent;
-    border-width: pxToRem(1);
+    border-width: 1px;
```

Comment:
This should be a separate commit from the other change.

Reply:
Ping me if you need me to split it up for you 🙂

Reply:
tnx. I fixed and pushed @adrianschmidt